docs/application/native/guides/device/overview.md

```diff
@@ -13,6 +13,10 @@ You can use the following device settings and systems features in your native ap
 
   You can control external output devices, allowing you to display the application screen on a separate device. You can manage the presentation mode, receive information and notifications from the device, and set the attributes for the external output.
 
+- [Diagnostics](diagnostics.md)
```

Comment:
A new page is added.
Please update docs/application/toc_all.md file also to location the new file.